controllers/user.js

```diff
@@ -4,23 +4,28 @@ const mysql   = require('anytv-node-mysql');
 const winston = require('winston');
 
 /**
- * @api {get} /user/:id Get user information
- * @apiName GetUser
+ * @api {post} /user Create new user
+ * @apiName CreateUser
  * @apiGroup User
  *
- * @apiParam {String} id User's unique ID
+ * @apiParam {String} email User's email
+ * @apiParam {String} password User's password
+ * @apiParam {String} first_name User's first name
+ * @apiParam {String} middle_initial User's middle initial
+ * @apiParam {String} last_name User's last name
  *
- * @apiSuccess {String} user_id User's unique ID
- * @apiSuccess {String} date_created Time when the user was created
- * @apiSuccess {String} date_updated Time when last update occurred
+ * @apiSuccess 
  */
-exports.get_user = (req, res, next) => {
-
+exports.create_user = (req, res, next) => {
+    
     function start () {
         mysql.use('my_db')
             .query(
-                'SELECT * FROM users WHERE user_id = ? LIMIT 1;',
-                [req.params.id],
+                'INSERT INTO users(email, password, first_name, middle_initial,\
+                                    last_name) \
+                 VALUES(?,?,?,?,?);',
+                [req.body.email, req.body.password, req.body.first_name,
```

Comment:
use util.get_data for getting data

https://github.com/AnyTV/anytv-node-boilerplate#creating-a-controller